CoinJoin.md

```diff
@@ -0,0 +1,129 @@
+
+Address Reuse: Users must adhere to the best practices for address generation. Those who keep reusing addresses may compromise their privacy.
+
+Amount correlation: If a user mixes an amount that stands out or is less common in the mix, it could be more easily traced, especially if this same amount appears in the next transactions. User's behavioral patterns, such as consolidating funds after mixing, could also provide clues to their identity.
+
+Non-Uniform adoption: The privacy and efficiency of CoinJoin also depend on how widely it is adopted. If only a small group of users are mixing transactions, it may be easier to identify those users identity.
+
+## CoinJoin Wallets
+
+CoinJoin is a Bitcoin-only protocol. The way the use of it differs depends on the wallet/mixer that you are using. Some of the most known privacy-focused Bitcoin wallets are Wasabi, Samourai. Let's go through some details concerning anonymity sets and fees for each of them:
+
+WabiSabi Wallet
+
+The WabiSabi Wallet charges a coordinator fee of 0.3% for CoinJoin transactions of more than 0.01 BTC. The coordinator fee is charged to cover the service costs of coordinating the mix. For amounts of 0.01 BTC or below, no coordinator fee is charged.
+
+An anonymity set is basically the pool of users participating in a specific transaction. The size of an ideal anonymity set in a WabiSabi can vary from 50 to 100 participants in a CoinJoin round. This means your transaction would be mixed with those 49 to 99 other users.
+
+Samourai 
+
+Samourai Wallet uses Whirlpool, a CoinJoin implementation created by its developer team. The service employs a fixed fee structure. Fees range from 5,000 satoshis for the smallest pool (100,000 satoshis) to 1,750,000 satoshis for the largest pool (50,000,000 satoshis). Each satoshi is equal to one hundred millionth of a Bitcoin (0.00000001 BTC), it is the smallest Bitcoin unit.
+
+Whirlpool's anonymity sets vary based on real-time participation within each pool. Since the number fluctuates the user is encouraged to go through multiple mixing rounds to increase their transaction's anonymity. Samourai wallet model aims to provide a balance between costs and privacy.
+
+Bitcoin mixers
+
+There are also options for those wanting a fast and user-friendly Bitcoin mixer. Websites like Whir and Tumber.io let you mix your Bitcoin by entering the Bitcoin address where you want to receive the mixed BTC. After that you need to send the BTC to the address generated by the tool, they will be sent to a BTC pool, where your Bitcoin will be mixed, and after that sent to the address you entered in the first step.
+
+Whir has a 1% fixed coordinator fee.
+Tumber.io fees start at 0.4% and vary according to the amount mixed.
+
+## Could it work for Decred?
+
+As we said before, CoinJoin is a tool exclusive to Bitcoin. Other coin mixers have been developed by other projects, like the Stake Shuffle protocol from Decred.
```

Comment:
"CoinJoin is a tool exclusive to Bitcoin" I don't think this is true.

Reply:
true, did a little more research and it said "To clarify, CoinJoin is not exclusive to Bitcoin. While it is most commonly associated with Bitcoin"